lib/services/history/item_history.ts

```diff
@@ -19,17 +19,19 @@ export class ItemHistory {
   constructor(entries?: ItemHistoryEntry[]) {
     /** Deserialize the entries into entry objects. */
     if (entries) {
-      for (const entry of entries) {
+      entries.slice(0).sort((a, b) => {
```

Comment:
What does slice(0) do? I could google it, but that's the point. Should be more clear.